### PR TITLE
Allow same private item to be added in the same page

### PR DESCRIPTION
**Pull request by leandrohmanica**

### CODE REVIEW

File: app/views/pages/_search_result.html.erb [hunk 1]
```diff
@@ -22,14 +22,9 @@
...
         </p>
-        <%# TODO: refactoring %>
-        <% if item.in_page? @page %>
-          <span class="already"><%= t 'views.item_already_in_this_page' %></span>
-        <% else @page %>
-          <%= button_to({action: 'add_to_page'}, remote: true,
-                params: {item: {id: item.id}}, class: 'button-box') do %>
-            <%= t 'views.add_to_page' %>
-          <% end %>
+        <%= button_to({action: 'add_to_page'}, remote: true,
```
Comment (longaraymatheus) [inline app/views/pages/_search_result.html.erb]: Does it make sense to leave/add `<% if @page %>` before adding the button tag?

Comment (leandrohmanica) [reply app/views/pages/_search_result.html.erb]: I'm not sure, I'm trying to imagine a scenario where it may break if @page isn't present.

**Comment by leandrohmanica** [comment]
it is really a bug?

**Comment by longaraymatheus** [comment]
@leandrohmanica not really right. Updated the label.

**Pull request merged**